roles/neutron-common/templates/etc/neutron/plugins/ml2/ml2_plugin.ini

```diff
@@ -19,6 +19,9 @@ tenant_network_types = {{ neutron.tenant_network_type }}
 {% endif %}
 mechanism_drivers = {{ mechanism_drivers|join(',') }}
 
+{% set extension_drivers = ['port_security'] %}
+extension_drivers = {{ extension_drivers|join(',') }}
```

Comment:
@j2sol 
So in neutron-common/defaults/main.yml add extension_drivers: port_security
and set that in template?

If envs need to overwrite, we can change in defaults-2.0.yml or all.yml?